gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -47,34 +49,12 @@ const renderChangeFacilityPartial = async ({ params, query, change, userToken })
     facilityId,
     status,
     change,
+    isFacilityEndDateEnabled: isFacilityEndDateEnabledOnGefVersion(parseDealVersion(deal.version)),
   };
 
   return body;
 };
 
-// returns body to render for template
-const renderBody = (body) => ({
-  facilityType: body.facilityType,
-  facilityName: body.facilityName,
-  hasBeenIssued: body.hasBeenIssued,
-  monthsOfCover: body.monthsOfCover,
-  shouldCoverStartOnSubmission: body.shouldCoverStartOnSubmission,
-  issueDateDay: body.issueDateDay,
-  issueDateMonth: body.issueDateMonth,
-  issueDateYear: body.issueDateYear,
-  coverStartDateDay: body.coverStartDateDay,
-  coverStartDateMonth: body.coverStartDateMonth,
-  coverStartDateYear: body.coverStartDateYear,
-  coverEndDateDay: body.coverEndDateDay,
-  coverEndDateMonth: body.coverEndDateMonth,
-  coverEndDateYear: body.coverEndDateYear,
-  facilityTypeString: body.facilityTypeString,
-  dealId: body.dealId,
-  facilityId: body.facilityId,
-  status: body.status,
-  change: body.change,
-});
```

Comment:
Where does this go / what's the replacement? (probs just me not following the diff well)